src/plus/integrations/providers/github.ts

```diff
@@ -317,3 +318,52 @@ export class GitHubEnterpriseIntegration extends GitHubIntegrationBase<SelfHoste
 		return super.connect(source);
 	}
 }
+
+export type GitHubPullRequestURLIdentity = {
+	ownerAndRepo?: string;
+	prNumber?: string;
+};
+
+export function getPullRequestIdentityValuesFromSearch(search: string): GitHubPullRequestURLIdentity {
+	let ownerAndRepo: string | undefined = undefined;
+	let prNumber: string | undefined = undefined;
+
+	let match = search.match(/github\.com\/([^/]+\/[^/]+)\/pull\/(\d+)/); // with github.com domain
+	if (match != null) {
+		ownerAndRepo = match[1];
+		prNumber = match[2];
+	}
+
+	if (prNumber == null) {
+		match = search.match(/([^/]+\/[^/]+)\/pull\/(\d+)/); // without domain
+		if (match != null) {
+			ownerAndRepo = match[1];
+			prNumber = match[2];
+		}
+	}
```

Comment:
Is there a reason for checking with the `github.com` domain first? The matching groups don't seem to be different, you could directly run the search without the domain, right? If the URL has github domain in it, it will work anyway with just the second query.
![Captura de pantalla 2024-10-21 a las 10 51 46](https://github.com/user-attachments/assets/d04af815-586c-4ebc-b0f5-70e937d659c3)

Reply:
@sergiolms 
correct. fixed.